wordpress-mobile/WordPress-Android
# Personalization accessibility

Fixes #21444 - As noted [here](https://github.com/wordpress-mobile/WordPress-Android/pull/21441#pullrequestreview-2427549531), our personalization cards are impossible to navigate with a screen reader. This PR aims to resolve that.

To test:

* At the bottom of the My Site tab, tap "Personalize your home tab"
* Enable TalkBack
* Note that the cards are clickable and the spoken labels are more descriptive

I also added a better `contentDescription` to the Shortcuts section.
